apps/expo-go/android/tools/src/main/java/host/exp/exponent/tools/ReactAndroidCodeTransformer.java

```diff
@@ -251,72 +259,41 @@ public Node visit(String methodName, MethodDeclaration n) {
         return n;
       }
     });
-    FILES_TO_MODIFY.put("devsupport/DevInternalSettings.java", new MethodVisitor() {
-
-      @Override
-      public Node visit(String methodName, MethodDeclaration n) {
-        switch (methodName) {
-          case "isReloadOnJSChangeEnabled":
-            BlockStmt blockStmt = JavaParser.parseBlock("{return false;}");
-            blockStmt.addOrphanComment(new LineComment(" NOTE(brentvatne): This is not possible to enable/disable so we should always disable it for"));
-            blockStmt.addOrphanComment(new LineComment(" now. I managed to get into a state where fast refresh wouldn't work because live reload"));
-            blockStmt.addOrphanComment(new LineComment(" would kick in every time and there was no way to turn it off from the dev menu."));
-            blockStmt.addOrphanComment(new LineComment(" return mPreferences.getBoolean(PREFS_RELOAD_ON_JS_CHANGE_KEY, false);"));
-            n.setBody(blockStmt);
-            return n;
-          case "setReloadOnJSChangeEnabled":
-            BlockStmt emptyBlockStmt = JavaParser.parseBlock("{}");
-            emptyBlockStmt.addOrphanComment(new LineComment(" NOTE(brentvatne): We don't need to do anything here because this option is always false"));
-            emptyBlockStmt.addOrphanComment(new LineComment(" mPreferences.edit().putBoolean(PREFS_RELOAD_ON_JS_CHANGE_KEY, enabled).apply();"));
-            n.setBody(emptyBlockStmt);
-            return n;
-          case "isHotModuleReplacementEnabled":
-            n.setPublic(true);
-            return n;
-          case "setHotModuleReplacementEnabled":
-            n.setPublic(true);
-            return n;
-          case "getPackagerConnectionSettings":
-            n.setPublic(true);
-            return n;
```

Comment:
WIth the Kotlinfy effort we no longer need to set these as public https://github.com/facebook/react-native/pull/43731